nncf/openvino/quantization/quantize_model.py

```diff
@@ -362,6 +365,8 @@ def quantize_with_accuracy_control_impl(
     """
     Implementation of the `quantize_with_accuracy_control()` method for the OpenVINO backend.
     """
+    model = remove_friendly_name_duplicates(model)
```

Comment:
Do you need this call?

Reply:
Yes, we need it. Why not?
We have the `quantize_with_tune_hyperparams` method inside the `quantize_with_accuracy_control_impl` along with `quantize_impl` that uses `model`.

Reply:
Does it mean that for AAQ there are many calls of `remove_friendly_name_duplicates` that does nothing? As at the first step it is called inside `quantize_with_accuracy_control_impl` and then inside `quantize_impl`

Reply:
Yes, it is. Do you have any other solution that would be more efficient in our pipeline?

Reply:
Not really. If there is no easy solution without refactoring all NNCF I would just drop a comment- something like: that there is overhead in AAQ of call `remove_friendly_name_duplicates`.